### ISSUE TITLE
Blog post for securing your Liberty application deployed using Open Liberty Operator using RedHat SSO

### ISSUE THREAD

**Issue by rumanaHaque** [OpenLiberty/blogs#3251]
Write a blog post about securing an Open Liberty application (deployed to OCP via the Open Liberty Operator) using RedHat SSO.

Security is paramount in today's world, and it is important to ensure your cloud applications are protected. A common way to do that is to configure Single SignOn for your application such as GitHub, Google, Facebook or OpenID Connect (OIDC). This presentation will show how to configure your Liberty application deployed in OpenShift to use a specific type of OIDC - RedHat Single SignOn (RH-SSO). Liberty application will be deployed using the Open Liberty Operator in OpenShift, with special configuration to be able to connect to RH-SSO. Configure the RH-SSO to create a client specifically for your liberty application. Once configuration is complete, when you login to your application, you will first be re-directed to RH-SSO, and once you authenticate successfully, it will redirect you back to your application.

**Comment by rumanaHaque** [comment]
Created a new branch from the prod - called https://github.com/OpenLiberty/blogs/tree/liberty-olo-with-RH-SSO

Making updates in the posts/2023-08-23-liberty-olo-rh-sso.adoc file.

**Comment by rumanaHaque** [comment]
Made updates to the 2023-08-23-liberty-olo-rh-sso.adoc file, and added these images
rh_social_media_guide.png
rh_social_media_logged_in.png
rh_social_media_login.png	
rh_social_media_redirect.png

Pushed it to the https://github.com/OpenLiberty/blogs/tree/liberty-olo-with-RH-SSO branch.

**Comment by rumanaHaque** [comment]
Created PR - https://github.com/OpenLiberty/blogs/pull/3389 - to check in to draft.

**Comment by rumanaHaque** [comment]
Merged the prev PR to draft site.

Created new PR to staging - https://github.com/OpenLiberty/blogs/pull/3390 - for review comments.

**Comment by rumanaHaque** [comment]
Incorporated the suggestions suggested in https://github.com/OpenLiberty/blogs/pull/3390 by Grace into my branch, and created pull request to draft to merge these changes in draft.
https://github.com/OpenLiberty/blogs/pull/3660
